src/App/Resources/AppResources.resx

```diff
@@ -584,7 +584,7 @@
     <value>A master password hint can help you remember your password if you forget it.</value>
   </data>
   <data name="MasterPasswordLengthValMessage" xml:space="preserve">
-    <value>Master password must be at least 8 characters long.</value>
+    <value>Master password must be at least 12 characters long.</value>
```

Comment:
It'd be good to change this resource to have the number of characters as a parameter as in `YourMasterPasswordCannotBeRecoveredIfYouForgetItXCharactersMinimum`. So then when we need to update the number we just update the `Constants` file.
Something like:
```xml
<data name="MasterPasswordLengthValMessageX" xml:space="preserve">
    <value>Master password must be at least {0} characters long.</value>
</data>
<!--Or IMO even better for reading and knowing exactly what the message is when looking the code outside of this file-->
<data name="MasterPasswordMustBeAtLeastXCharactersLong" xml:space="preserve">
    <value>Master password must be at least {0} characters long.</value>
</data>
```